xstream/server/server/ruleManager.go

```diff
@@ -69,6 +69,7 @@ func doStartRule(rs *RuleState) error {
 				tp.GetContext().SetError(err)
 				logger.Printf("closing rule %s for error: %v", rs.Name, err)
 				tp.Cancel()
+				rs.Triggered = false
```

Comment:
Is this done in another PR?

Reply:
no The state of the rule needs to be changed when the rule is stopped